.github/ISSUE_TEMPLATE/hosting.md

```diff
@@ -0,0 +1,57 @@
+---
+name: hosting
+about: Wanting to host the meetup? Answer the following questions
+---
+
+Thank you for wanting to host the Ruby Usergroup Hamburg!
+Here we have collected some information regarding hosting the Usergroup.
+Additionally, it would be great if you could answer the questions at the end of the issue. This will help us to have a good overview
+
+# FAQ
+
+## When can I host the meetup
+We meet every **second Wednesday** of the month
+Any [event](https://hamburg.onruby.de/#events) where no location is named still requires one
+
+## How much space do we need?
+There have been **17** Rubyists signed up on average for the latest events and there were about **1** to **2** presentations held. 
+
+Those numbers are just guidelines, make sure to talk to one of the [organizers](https://hamburg.onruby.de/#topics) when preparing the hosting.
+
+If you don't have a lot of space, it's also possible to *limit* the number of participants.
+
+## What's needed for a Usergroup?
+The location, of course, but there is more to it:
+ * alcohol-free beverages
+ * beamer and adapters
+ * beer (optional)
+ * snacks/pizza (optional)
+
+You don't need to make a fuss, we are cool with the bare minimum!
+We believe everyone should be able to host the meetup. A limited budget, because you are an NGO or a small company, should not be a stopper.
+
+
+# Planning
+
+## Who is the person we can contact in case of questions?
+
+
+## When do you want to host us?
+{Date of the meetup} {Time of the meetup}
+
+## Is the location address still update to date?
+On our [website](https://hamburg.onruby.de/locations) we offer directions to the location. Please check if your address is still correct: 
+[ ] Yes
+[ ] No (please comment the correct address below)
+
+## What do you want to offer?
+[x] alcohol free beverages
+[ ] beer (optional)
+[ ] snacks (optional) 
+[ ] dinner (pizza, pretzels, etc.) (optional)
+
+## Will you need some compensation?
```

Comment:
what is the purpose of this question?

Reply:
The motivation is to be clear that anyone could host the meetup and we are not expecting them to cover all costs. Instead, we could also have a collection box somewhere where people could donate money.

Yet I also see that this section might be confusing and will remove it.

Reply:
that has never happened and i would strongly disagree with payments aka donating money in order to participate at an event.